apps/signing/serializers.py

```diff
@@ -0,0 +1,81 @@
+import json
+
+from django.core.urlresolvers import reverse
+
+from rest_framework import serializers
+
+import amo
+from amo.helpers import absolutify
+from files.models import FileUpload
+
+
+class FileUploadSerializer(serializers.ModelSerializer):
+    active = serializers.SerializerMethodField('get_active')
+    api_url = serializers.SerializerMethodField('get_api_url')
+    files = serializers.SerializerMethodField('get_files')
+    passed_review = serializers.SerializerMethodField('get_passed_review')
+    processed = serializers.CharField(source='processed')
```

Comment:
Interestingly it is actually returned as a boolean in the response even though it's marked as a `CharField`.